docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
```

Comment:
Missing documentation for:
- OTEL_TRACE_CONFIG_FILE
- OTEL_VERSION
- ...

Please double-check everything with: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/src/Datadog.Trace/Configuration/ConfigurationKeys.cs 😉